### ISSUE TITLE
The bottom of the intro wizard not visible when starting the application on computers with small screens

### ISSUE THREAD

**Issue by SunyataZero** [mindfulness-at-the-computer/mindfulness-at-the-computer#317]
Can be fixed easily by changing the y coordinate for the window. You can find this line in the file mc/gui/intro_dlg.py:
`self.setGeometry(300, 300, 650, 500)`

The second coordinate is the y coordinate which we could change to 150 for example

**Comment by fipnova51** [comment]
Can I take this issue?
It'd be my first contribution to github...

**Comment by SunyataZero** [comment]
@fipnova51 Yes, please do! And welcome :)

**Comment by fipnova51** [comment]
@SunyataZero ,

Sorry but I couldn't see the the issue to fix. It seems there's a visibility issue on small screen.

I did some tests bu changing line 63 of files mc\gui\intro_dlg.py

I tested with

'''
    self.setGeometry(3000, 3000, 650, 500)
    self.setGeometry(300, 300, 6500, 5000)
'''

But every time I execute `python mindfulness-at-the-computer.py` I didn't see any changes in the gui (testing on Windows 7)

I'm ok to commit the change `self.setGeometry(300, 150, 650, 500)` but I'd have prefered to see the bug and correct it.

Am I doing anything wrong?
